cmd/fleetctl/mdm.go

```diff
@@ -122,13 +122,9 @@ func mdmRunCommand() *cli.Command {
 				hostUUIDs = append(hostUUIDs, host.UUID)
 			}
 
-			if len(hostUUIDs) == 0 {
-				// all hosts were not found
-				return errors.New("No hosts targeted. Make sure you provide a valid hostname, UUID, osquery host ID, or node key.")
-			}
-			if notFoundCount > 0 {
-				// at least one was not found
-				return errors.New("One or more targeted hosts don't exist. Make sure you provide a valid hostname, UUID, osquery host ID, or node key.")
+			if len(hostUUIDs) == 0 || notFoundCount > 0 {
+				// Either no hosts were targeted, or at least one targeted host was not found
+				return errors.New(fleet.TargetedHostsDontExistErrMsg)
```

Comment:
(For the future) we should somehow tell the user which hosts were not found...

Reply:
cc @marko-lisica